.github/workflows/python.yml

```diff
@@ -0,0 +1,39 @@
+name: Python CI
+
+on:
+  push:
+    branches:
+     - '*'
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: build-${{ matrix.python-version }}
+    runs-on: "ubuntu-latest"
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+      with:
+        submodules: true
+
+    - name: deps
+      run: sudo apt-get update && sudo apt-get install libxml2-utils gettext
```

Comment:
```suggestion
      run: sudo apt update -qq && sudo apt install -yq libxml2-utils gettext
```

Reply:
I dropped the update, it works without that.